Ukol_2/cpp/sifry.cpp

```diff
@@ -1,42 +1,175 @@
 #include <iostream>
 #include <fstream>
 #include <string>
+#include <sstream>
+#include <iomanip>
 
+using namespace std;
 // Funkce pro otevření souboru
 std::string otevri_soubor(const std::string &jmeno_souboru)
 {
   // Implementace funkce pro otevření souboru a načtení jeho obsahu
-  return "";
+  ifstream soubor_vstup(jmeno_souboru);
+  if (!soubor_vstup.is_open()) {
+    std::cerr << "Failed to open the file!" << std::endl;
+    return;
```

Comment:
Váš kompilátor je schopný sestavit tento program? U mě tento řádek neakceptuje protože return musí mít nějakou hodnotu i kdyby to měl být prázdný string `return ""`